en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -300,27 +300,27 @@ The index map for covariances\[28\] is as follows:
 
 ### Observation Innovations
 
-* Magnetometer XYZ \(gauss\) : Refer to mag\_innov\[3\] in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg).
-* Yaw angle \(rad\) : Refer to heading\_innov in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg).
-* Velocity and position innovations : Refer to vel\_pos\_innov\[6\] in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg). The index map for vel\_pos\_innov\[6\] is as follows:
+* Magnetometer XYZ \(gauss\) : Refer to `mag_field[3]` in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg).
+* Yaw angle \(rad\) : Refer to `heading` in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg).
+* Velocity and position innovations : Refer to vel\_pos\_innov\[6\] in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg). The index map for vel\_pos\_innov\[6\] is as follows:
   * \[0 ... 2\] Velocity NED \(m/s\)
   * \[3 ... 5\] Position NED \(m\)
-* True Airspeed \(m/s\) : Refer to airspeed\_innov in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg).
-* Synthetic sideslip \(rad\) : Refer to beta\_innov in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg).
-* Optical flow XY \(rad/sec\) : Refer to flow\_innov in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg).
-* Height above ground \(m\) : Refer to hagl\_innov in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg).
+* True Airspeed \(m/s\) : Refer to `airspeed` in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg).
+* Synthetic sideslip \(rad\) : Refer to `beta` in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg).
+* Optical flow XY \(rad/sec\) : Refer to `flow` in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg).
+* Height above ground \(m\) : Refer to `hagl` in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg).
 
 ### Observation Innovation Variances
 
-* Magnetometer XYZ \(gauss^2\) : Refer to mag\_innov\_var\[3\] in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg).
+* Magnetometer XYZ \(gauss^2\) : Refer to mag\_innov\_var\[3\] in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg). 
```

Comment:
@priseborough All of these items missing from estimator_innovations (the _innov_var variants). Where are these now?

Reply:
@mhkabir Is this something you can advise on? ^^^

Reply:
@mhkabir ^^^^ There are a couple off open questions on this PR. Can you help?